jme3-core/src/main/java/com/jme3/anim/SeparateJointModelTransform.java

```diff
@@ -30,9 +30,12 @@ public void getOffsetTransform(Matrix4f outTransform, Matrix4f inverseModelBindM
 
     @Override
     public void applyBindPose(Transform localTransform, Matrix4f inverseModelBindMatrix, Joint parent) {
-        localTransform.fromTransformMatrix(inverseModelBindMatrix.invert());
+        Matrix4f safeInverseBind = inverseModelBindMatrix.clone().invert(); // avoid mutating input
```

Comment:
The `Matrix4f.invert()` method already creates a new object, so there is no need to clone the input matrix before inverting it.